src/utils/redux/saga/index.js

```diff
@@ -0,0 +1,11 @@
+import { all, fork } from 'redux-saga/effects';
+
+import pages from '../../../pages/data/sagas';
+
+const sagas = [
+  ...pages,
+];
+
+export default function* rootSaga() {
+  yield all(sagas.map(saga => fork(saga)));
```

Comment:
You don't need to fork inside the `all` effect, actually `all` is a high-level effect that uses `fork` internally.